brix11/lib/brix11/formatter.rb

```diff
@@ -163,7 +163,7 @@ def initialize(filters, verbosity = 1)
 
       # override
       def print(*args)
-        txt = (@cache.to_s << args.flatten.join)
+        txt = (@cache.to_s.dup << args.flatten.join)
```

Comment:
This unnecessarily duplicates strings. Use `((@cache || '') << args.flatten.join)` instead